code/02_create-figures.R

```diff
@@ -0,0 +1,310 @@
+source(here::here("functions", "summarise_data.R"))
+
+
+### 2 - Read in basefile ----
+
+basefile <- read_rds(here("data", "basefiles", 
+                          glue("{pub_date}_base-file.rds")))
+
+
+### 3 - Figure 1 - Trend Bar Chart ----
+
+fig1 <- 
+  
+  basefile %>%
+  summarise_data(include_years = "all",
+                 format_numbers = FALSE) %>%
+  pivot_longer(cols = qom:qom_hosp,
+               names_to = "qom") %>%
+  mutate(qom = if_else(qom == "qom_hosp",
+                       "Hospital",
+                       "Home/Community")) %>%
+  
+  # Control order of stacks in chart
+  mutate(qom = fct_relevel(as.factor(qom), "Hospital")) %>%
+  
+  ggplot(aes(x = fy, y = value, fill = qom)) +
+  geom_bar(position = "stack", stat = "identity", width = 0.5, show.legend = T) +
+  scale_fill_manual(values = c("#00a2e5", "#004785")) +
+  theme(panel.background = element_blank(),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        axis.title.x = element_text(size = 10, face = "bold"),
+        axis.title.y = element_text(size = 10, face = "bold"),
+        axis.text = element_text(size = 10),
+        axis.text.x = element_text(angle = 45),
+        legend.title = element_blank()) +
+  xlab("Financial Year of Death") + 
+  ylab("Percentage")
+
+ggsave(here("markdown", "figures", "figure-1.png"), 
+       plot = fig1,
+       width = 17.49, height = 9.03, 
+       units = "cm", device = "png", dpi = 600)
+
+ggsave(here("markdown", "figures", "figure-1-summary.png"), 
+       plot = fig1,
+       width = 17, height = 7, 
+       units = "cm", device = "png", dpi = 600)
+
+
+### 4 - Figure 2 - Health Board Map ----
+
+fig2 <- shapefile()
+
+fig2@data %<>%
+  left_join(summarise_data(basefile, hb,
+                           format_numbers = FALSE) %>% 
+              mutate(hb = substring(hb, 5)) %>%
+              select(hb, qom),
+            by = c("HBName" = "hb"))
+
+fig2@data$id <- rownames(fig2@data)
+
+fig2 <-
+  full_join(tidy(fig2, region = "id"),
+            fig2@data,
+            by = "id")
+
+fig2 <-
+  
+  ggplot() +
+  geom_polygon(data = fig2,
+               aes(x = long,
+                   y = lat,
+                   group = group,
+                   fill = qom),
+               colour = "white",
+               size = 0.3) +
+  scale_fill_continuous(trans = "reverse") +
+  theme(panel.background = element_blank(),
+        panel.grid = element_blank(),
+        axis.text = element_blank(),
+        axis.title = element_blank(),
+        axis.ticks = element_blank(),
+        legend.title = element_blank())
+
+ggsave(here("markdown", "figures", "figure-2.png"), 
+       plot = fig2,
+       width = 10, height = 12, 
+       units = "cm", device = "png", dpi = 600)  
+
+
+### 5 - Figure 3 - Age/Sex Bar Chart ----
+
+fig3 <- 
+  
+  basefile %>%
+  filter(!is.na(sex)) %>%
+  summarise_data(age_grp, sex, format_numbers = FALSE) %>%
```

Comment:
I noticed you have bars for "All Ages" in the last publication report. Do we need to include this?